dash/development/base_component.py

```diff
@@ -550,6 +758,25 @@ def parse_wildcards(props):
             list_of_valid_wildcard_attr_prefixes.append(wildcard_attr[:-1])
     return list_of_valid_wildcard_attr_prefixes
 
+def parse_wildcards_r(props):
```

Comment:
:camel: I do not think this function is necessary, it is the same as `parse_wildcards` currently. We can keep this as one function used by the R and Python generation until we have a reason to re-write it.